sign/options.go

```diff
@@ -101,6 +101,8 @@ func Default() *Options {
 		MaxWorkers:           100,
 		CacheTimeout:         2 * time.Hour,
 		MaxCacheItems:        10000,
+		CertIdentityRegexp:   "(prow-build|krel-trust|krel-staging)@k8s-infra-prow-build.iam.gserviceaccount.com",
```

Comment:
the krel-trust is this email `"krel-trust@k8s-releng-prod.iam.gserviceaccount.com`